extensions/query/events/system/email/email_delivery.json

```diff
@@ -0,0 +1,33 @@
+{
+  "description": "Email Delivery events report the delivery status of emails.",
+  "extends": "email_activity",
+  "caption": "Email Delivery Activity",
+  "name": "email_delivery_activity",
```

Comment:
Remind me, this is because we decided not to backport `EmailActivity` from the more recent versions of OCSF and instead add some of its attributes to `EmailDelivieryActivity` which exists in our version?

Reply:
OCSF has three different event classes for email activity, viz, Email Activity, Email Url Activity (an extra url attr) and Email File Activity (an extra file attr). We had decided to use Email Activity by adding these two attributes to Email Activity itself, so that we normalize all email activities to same event class. We don't need to extend if we use the other two as well, but it would mean more work in config, normalization, lengthier queries

https://queryai.atlassian.net/browse/QAI-283?focusedCommentId=22761